docs/defaults-detailed.rst

```diff
@@ -334,6 +334,26 @@ Firewall parameters
   ``True``, the role will generate configuration for the debops.ferm_ Ansible
   role to enable packet forwarding for a given interface.
 
+``forward_interface_ferm_rule_enabled``
+  Optional, boolean. Should a Firewall rule be configured which matches new
+  connection attempts entering the interface?
+  If disabled using ``False``, the default Firewall policy will apply.
+  Defaults to ``True``.
```

Comment:
Since the default firewall policy applies when the custom ferm rules are disabled, I would make the default as `False` for both of them.